manual/cops_style.md

```diff
@@ -1522,6 +1522,39 @@ redundant.
 -> (arg) { do_something(arg) }
 ```
 
+## Style/EmptyLineAfterGuardClause
+
+Enabled by default | Supports autocorrection
+--- | ---
+Disabled | Yes
+
+This cop enforces empty line after guard clause
+
+### Examples
+
+```ruby
+# bad
+def foo
+  return if need_return?
+  bar
+end
+
+# good
+def foo
```

Comment:
You should also show an example with consecutive guard clauses.

Reply:
@bbatsov example added. The build now is failed, but it seems, that it is broken in master.